sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/jdbc/AbstractAzureJdbcAutoConfigurationTest.java

```diff
@@ -63,6 +67,7 @@ void testHasSingleBean() {
     void testNoAzureAuthenticationTemplate() {
         this.contextRunner
             .withClassLoader(new FilteredClassLoader(AzureAuthenticationTemplate.class))
+            .withConfiguration(AutoConfigurations.of(AzureGlobalProperties.class))
```

Comment:
the contextRunner already contains the `AzureGlobalPropertiesAutoConfiguration`, do we need this line here?